composer.json

```diff
@@ -14,7 +14,8 @@
         }
     },
     "require": {
-        "amphp/artax": "^3",
+        "php": "^7.0",
```

Comment:
Right, and you don't support 7.2 either strictly speaking, because it isn't in feature freeze yet. There are _some_ BC breaks in every version of PHP. What's the purpose of restricting the upper bound? Most code will just run fine. As long as PHP itself isn't installed via Composer, I don't see any reason to restrict the upper bound, which will mostly just work fine for new versions.